internal/stores/hostdb_test.go

```diff
@@ -167,13 +176,17 @@ func TestSQLHostDB(t *testing.T) {
 		t.Fatal("known since not set")
 	}
 
-	// Apply a consensus change to make sure the ccid is updated.
+	// Wait for the persist interval to pass to make sure an empty consensus
+	// change triggers a persist.
+	time.Sleep(2 * time.Second)
```

Comment:
`nit` the persist interval used in testing could be a constant

I'm also not sure how waiting for the duration of the persist interval makes sure an empty CC triggers a persist? You're not really asserting anything here right?

Reply:
A few lines further down we are calling `ProcessConsensusChange` which then checks for the time since the last persist. Afterwards we are creating the database again and if we don't wait here, `NewSQLStore` returns a wrong `ccid`.